src/report.tex

```diff
@@ -1182,16 +1182,9 @@ \subsection{Neutron Transport}
 Initially calculations were batched into groups of $5\times10^7$ particles per
 batch, however these calculations did not finish in a timely enough manner and
 were instead reduced to groups of $5\times10^6$ particles per batch. With the
-smaller batching system calculations did finish, however, only 10\% of all
+smaller batching system calculations did finish, however, only 5\% of all
```

Comment:
I'll rework this.  We will NOT have any description of what fraction of jobs completed.  It is irrelevant, distracting, and clearly a source of challenge.  We only need to describe how many histories we did use.